programs/svm-spoke/src/instructions/bundle.rs

```diff
@@ -81,6 +81,11 @@ impl RelayerRefundLeaf {
     pub fn to_bytes(&self) -> Vec<u8> {
         let mut bytes = Vec::new();
 
+        // This requires the first 64 bytes to be 0 within the encoded leaf data. This protects any kind of EVM leaf
```

Comment:
Maybe we can expand a bit here where 64 bytes come from, like 2 uint256 fields for EVM amountToReturn and chainId